src/index.ts

```diff
@@ -79,6 +79,14 @@ export interface IUpdateElectronAppOptions<L = ILogger> {
 const pkg = require('../package.json');
 const userAgent = format('%s/%s (%s: %s)', pkg.name, pkg.version, os.platform(), os.arch());
 const supportedPlatforms = ['darwin', 'win32'];
+const isHttpsUrl = (maybeURL: string) => {
+  try {
+    const url = new URL(maybeURL);
+    return url.protocol === 'https:';
```

Comment:
```suggestion
    const { protocol } = new URL(maybeURL);
    return protocol === 'https:';
```

nit, but more closely aligns with [the similar code we have in Fiddle](https://github.com/electron/fiddle/blob/3452e12a67682aba8898019d1ae07df8e3194d61/src/main/windows.ts#L27-L34).